### PR TITLE
Spring cleanup

### PR DESCRIPTION
**Pull request by FilipDolnik**

### CODE REVIEW

File: CuckooGenerator.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -499,7 +631,7 @@
 				GCC_OPTIMIZATION_LEVEL = 0;
 				GCC_PREPROCESSOR_DEFINITIONS = (
 					"DEBUG=1",
-					"$(inherited)",
+					"PROJECT_DIR=@\\\"$(PROJECT_DIR)\\\"",
```
Comment (TadeasKriz) [inline CuckooGenerator.xcodeproj/project.pbxproj]: What?

File: CuckooGenerator.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -499,7 +631,7 @@
 				GCC_OPTIMIZATION_LEVEL = 0;
 				GCC_PREPROCESSOR_DEFINITIONS = (
 					"DEBUG=1",
-					"$(inherited)",
```
Comment (TadeasKriz) [inline CuckooGenerator.xcodeproj/project.pbxproj:502]: Should not be removed.

File: CuckooGenerator.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -513,6 +645,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				OTHER_CODE_SIGN_FLAGS = "--deep";
 				SDKROOT = macosx;
+				STRIP_INSTALLED_PRODUCT = YES;
```
Comment (TadeasKriz) [inline CuckooGenerator.xcodeproj/project.pbxproj]: Why?

File: Source/CuckooGeneratorFramework/Api/GenerateMocksCommand.swift [hunk 4]
```diff
@@ -10,15 +10,17 @@ import Commandant
...
+public struct GenerateMocksCommand: CommandType {
     
-    let verb = "generate"
-    let function = "Generates mock files"
+    public let verb = "generate"
+    public let function = "Generates mock files"
     
-    func run(options: Options) -> Result<Void, CuckooGeneratorError> {
-        let parsedFiles = options.files.map { File(path: $0) }.filterNil().map { options.runtime.tokenizer.init(sourceFile: $0).tokenize() }
+    public init() {
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/Api/GenerateMocksCommand.swift]: Does it have to have an explicit init?

File: Source/CuckooGeneratorFramework/Tokenizers/Tokenizer_r1.swift [hunk 5]
```diff
@@ -27,7 +27,7 @@ struct Tokenizer_r1: Tokenizer {
     }
     
     private func tokenize(representables: [SourceKitRepresentable]) -> [Token] {
-        return representables.map(tokenize).filterNil()
+        return representables.map(tokenize).flatMap { $0 }
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/Tokenizers/Tokenizer_r1.swift]: ?

Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/Tokenizers/Tokenizer_r1.swift]: Infinite recursion?

File: Source/CuckooGeneratorFramework/Tokenizers/Tokenizer_r1.swift [hunk 6]
```diff
@@ -150,7 +150,7 @@ struct Tokenizer_r1: Tokenizer {
         let parameters = methodName.componentsSeparatedByString("(").last?.characters.dropLast(1).map { "\($0)" }.joinWithSeparator("")
         let parameterLabels: [String?] = parameters?.componentsSeparatedByString(":").map { $0 != "_" ? $0 : nil } ?? []
         
-        return zip(parameterLabels, representables).map(tokenizeMethodParameter).filterNil()
+        return zip(parameterLabels, representables).map(tokenizeMethodParameter).flatMap { $0 }
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/Tokenizers/Tokenizer_r1.swift]: Why do you keep removing the `filterNil()`?

File: Source/CuckooGeneratorFramework/Supporting Files/CuckooGeneratorFramework.h [hunk 7]
```diff
@@ -16,4 +16,4 @@ FOUNDATION_EXPORT const unsigned char CuckooGeneratorFrameworkVersionString[];
 
 // In this header, you should import all the public headers of your framework using statements like #import <CuckooGeneratorFramework/PublicHeader.h>
 
-
+#define PROJECT_DIR PROJECT_DIR
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/Supporting Files/CuckooGeneratorFramework.h]: This is an exploit and will not work in the future AFAIK.

Comment (FilipDolnik) [inline Source/CuckooGeneratorFramework/Supporting Files/CuckooGeneratorFramework.h]: Will be removed with the introduction of Aruba.

**Comment by TadeasKriz** [comment]
👍

**Pull request merged**